### PR TITLE
Add youtube-dl

### PR DESCRIPTION
**Pull request by balrog-nona**
Requires https://github.com/OpenIndiana/oi-userland/pull/5568

### CODE REVIEW

File: components/multimedia/youtube-dl/Makefile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_TMP_VERSION).tar.gz
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/downloads/latest/$(COMPONENT_ARCHIVE)	
+COMPONENT_ARCHIVE_HASH= \
+	sha256:0b1dcc96d204984df1ee364e81cddd5645421f6065d2815472258ccdad5c36ff
+COMPONENT_LICENSE=	Unlicense
+COMPONENT_LICENSE_FILE=	LICENSE
+
+PYTHON_VERSIONS=	3.5
+
+include $(WS_MAKE_RULES)/common.mk
```
Comment (Mno-hime) [inline components/multimedia/youtube-dl/Makefile:38]: Add `TEST_TARGET=$(NO_TESTS)` above `common.mk` include if there is no test suite.

Comment (balrog-nona) [reply components/multimedia/youtube-dl/Makefile:38]: Did it.

**Comment by alarcher** [comment]
@Mno-hime Can we merge this PR?

File: components/multimedia/youtube-dl/youtube-dl.p5m [hunk 2]
```diff
@@ -0,0 +1,833 @@
...
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Nona Hansel
+#
+
+#<transform file -> add pkg.depend.bypass-generate usr/lib/libgcc_s\\.so\\.1 >
```
Comment (pyhalov) [inline components/multimedia/youtube-dl/youtube-dl.p5m]: Please, remove commented line.

File: components/multimedia/youtube-dl/Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		youtube-dl
+COMPONENT_VERSION=	2020.1.24
+COMPONENT_TMP_VERSION=	2020.01.24
+COMPONENT_SUMMARY=	'Command-line program to download videos from YouTube.com and other video sites'
+COMPONENT_PROJECT_URL=	https://youtube-dl.org/
+COMPONENT_FMRI=		multimedia/video/youtube-dl
```
Comment (pyhalov) [inline components/multimedia/youtube-dl/Makefile]: Please, avoid introducing new FMRI 'tree'  multimedia/video/... We have video/* and multimedia/*

File: components/multimedia/youtube-dl/youtube-dl.p5m (hunk 2 above)
Comment (balrog-nona) [reply components/multimedia/youtube-dl/youtube-dl.p5m]: Did it.

File: components/multimedia/youtube-dl/Makefile (hunk 3 above)
Comment (balrog-nona) [reply components/multimedia/youtube-dl/Makefile]: I've chosen multimedia/youtube-dl

File: components/multimedia/youtube-dl/Makefile [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+BUILD_BITS=		NO_ARCH
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		youtube-dl
+COMPONENT_VERSION=	2020.3.8
+COMPONENT_TMP_VERSION=	2020.03.08
+COMPONENT_SUMMARY=	'Command-line program to download videos from YouTube.com and other video sites'
```
Comment (pyhalov) [inline components/multimedia/youtube-dl/Makefile]: As PYV rules are not used, COMPONENT_SUMMARY is not transformed twice and ' signs appear in pkg summary. Please, remove ' from this line..

**Comment by pyhalov** [comment]
Otherwise, looks good.

File: components/multimedia/youtube-dl/Makefile (hunk 4 above)
Comment (balrog-nona) [reply components/multimedia/youtube-dl/Makefile]: Removed, thanks.

**Pull request merged**